modules/diskq/qdisk.c

```diff
@@ -343,8 +341,7 @@ _compat_preallocate(QDisk *self, gint64 size)
       return FALSE;
     }
 
-  self->cached_file_size = pos + additional_write_size;
-  g_assert(self->cached_file_size == size);
+  g_assert(pos + additional_write_size == len + offset);
```

Comment:
`len + offset` are undeclared in this commit.

Reply:
Yes, I messed up the commit order. Fixed. :)